modules/porous_flow/doc/content/bib/porous_flow.bib

```diff
@@ -487,3 +487,46 @@ @book{javendel
 publisher = {AGU},
 year = {1984}
 }
+
+@article{doughty2007,
+author = {C. Doughty},
+year = {2007},
+pages = {1768--1781},
+title = {Modeling geologic storage of carbon dioxide: Comparison of non-hysteretic and hysteretic characteristic curves},
+volume = {48},
+journal = {Energy Conservation and Management},
+doi = {10.1016/j.enconman.2007.01.022}
+}
+
+@techreport{doughty2008,
+  title={User's guide for hysteretic capillary pressure and relative permeability functions in {iTOUGH2}, Rev3, Draft, March 2008},
```

Comment:
The March renders with a lowercase m on the website

Reply:
Thanks